cloudsmith/data_source_package.go

```diff
@@ -95,6 +112,30 @@ func downloadPackage(url string, downloadDir string, pc *providerConfig) (string
 	return outputPath, nil
 }
 
+func calculateChecksums(filePath string) (string, string, string, string, error) {
```

Comment:
nit picking but returning 5 values probably grants creating a `struct` for readability.